Gradata/tests/test_rule_graduated_events.py

```
@@ -0,0 +1,179 @@
+INSTINCT/PATTERN/RULE/UNTESTABLE/KILLED transition emits a RULE_GRADUATED
+event with {category, description, old_state, new_state, confidence,
+fire_count, reason}.
+"""
+
+from __future__ import annotations
+
+import json
+from pathlib import Path
+
+from gradata._types import Lesson, LessonState
+from gradata.brain import Brain
```

Ruff (0.15.10)

[warning] Move application import `gradata.brain.Brain` into a type-checking block

Move into type-checking block

(TC001)